test/llvmpasses/refinements.ll

```diff
@@ -0,0 +1,55 @@
+; RUN: opt -load libjulia.so -LateLowerGCFrame -S %s | FileCheck %s
```

Comment:
should we be trying to run these on CI eventually? they don't take long to run do they?